website/static/js/osfLanguage.js

```diff
@@ -80,5 +80,10 @@ module.exports = {
         dataSendError: 'Error sending data to the server: check that all fields are valid, or contact ' + SUPPORT_LINK + ' if the problem persists.',
         creationSuccess: 'Successfully registered new application',
         dataUpdated: 'Application data updated'
+    },
+    ProjectSettings: {
```

Comment:
Since this is closer to a variable than a class name, please use lowerCamelCase:

https://github.com/felixge/node-style-guide#use-lowercamelcase-for-variables-properties-and-function-names